Code.Rmd

```diff
@@ -173,25 +412,27 @@ For now, you should always use `.onLoad()` unless explicitly directed otherwise.
 
 Some common uses of `.onLoad()` and `.onAttach()` are:
 
-*   To display an informative message when the package loads. This might make 
-    usage conditions clear, or display useful tips. Startup messages is one 
-    place where you should use `.onAttach()` instead of `.onLoad()`. To display 
-    startup messages, always use `packageStartupMessage()`, and not `message()`. 
-    (This allows `suppressPackageStartupMessages()` to selectively suppress 
-    package startup messages).
+* To display an informative message when the package loads. This might make 
```

Comment:
BTW I prefer greater indenting here so that the code and bullets line up nicely.